backstop/tests/tabs.html

```diff
@@ -18,247 +18,843 @@
-      velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-      cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
-      est laborum.
-    </div>
+    <h2>Default</h2>
+    <section id="demo-default">
 
-    <br />
+      <ul
+        class="iui-tabs iui-default"
+        role="tablist"
+        aria-label="Tab group name"
+      >
+        <li>
+          <button
+            class="iui-tab iui-active"
+            role="tab"
+            aria-selected="true"
+            aria-controls="active-tab"
+            tabindex="-1"
+          >
+            <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-tab-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
+            >
+              <path
+                d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+              />
+            </svg>
+            <span class="iui-tab-label">Active</span>
+          </button>
+        </li>
+        <li>
+          <button
+            class="iui-tab"
+            role="tab"
+            aria-selected="false"
+            aria-controls="left-tab"
+          >
+            <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-tab-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
+            >
+              <path
+                d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+              />
+            </svg>
+            <span class="iui-tab-label">Left</span>
+          </button>
+        </li>
+        <li>
+          <button
+            class="iui-tab"
+            role="tab"
+            aria-selected="false"
+            aria-controls="disabled-tab"
+            disabled
+          >
+            <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-tab-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
+            >
+              <path
+                d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+              />
+            </svg>
+            <span class="iui-tab-label">Disabled</span>
+          </button>
+        </li>
+        <li>
+          <button
+            class="iui-tab"
+            role="tab"
+            aria-selected="false"
+            aria-controls="none-tab"
+          >
+            <span class="iui-tab-label">None</span>
+          </button>
+        </li>
+        <li>
+          <button class="iui-button iui-borderless iui-small">
```

Comment:
Why not using the same tab styling with icon?
![image](https://user-images.githubusercontent.com/36186912/122181736-4dc16880-ce92-11eb-81d1-f8dcabb0f8bb.png)

Reply:
The plus button I have currently is not in any standards nor has it been requested by any devs.  I was trying to think through "should the tabs use the baseline or input-height variables".  Here's another example of placing an input (select) inline with the tabs where the tabs are using baseline (44px) and the input is using the input-height (38px).
![Screen Shot 2021-06-16 at 12 02 15 PM](https://user-images.githubusercontent.com/849817/122254271-f7a4f300-ce9a-11eb-9eef-e30667ac848a.png)

I am still questioning which variables we should use for the tabs.

Reply:
Decided against baseline, using the input height variables for all tabs.